src/Root.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {
+  HashRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import { App } from './App';
+import { PeoplePage } from './components/PeoplePage';
+
+export const Root: React.FC = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<h1 className="title">Home Page</h1>} />
+        <Route path="/home" element={<Navigate to="/" />} />
```

Comment:
I think better to add replace attribute not to save /home URL in the browser history and avoid the navigation loop when you press the browser Go back button.